views/js/controller/Delivery/monitoring.js

```diff
@@ -452,6 +448,78 @@ define([
                 }
 
                 /**
+                 * Create a warning message for execBulkAction()
+                 * @param {String} action
+                 * @param {String[]} selection
+                 * @param {Object[]} deniedResources
+                 */
+                function buildWarningMessage(action, selection, deniedResources) {
+                    var maxReasons = 2;
+                    var warningAction;
+                    var warningReason;
+                    var warningReasonOmission = __('...');
+
+                    deniedResources = deniedResources || [];
+                    selection = selection || [];
+
+                    warningAction = getWarningAction(action, deniedResources.length > 1 || selection.length > 1);
+                    warningReason = _(deniedResources || [{}])
+                        .slice(0, maxReasons)
+                        .map(function (deniedResource, index) {
+                            return getWarningReason(deniedResource.reason, deniedResource.id || (index + 1));
+                        })
+                        .join(' ');
+
+                    return __('%s %s%s', warningAction, warningReason, (deniedResources.length > maxReasons ? warningReasonOmission : ''));
```

Comment:
I think there is no need to use the translate helper to build such a string: there is nothing to translate, just a few placeholders. So you should just concat the strings.